doc/source/api/api_contents.rst

```diff
@@ -56,5 +56,5 @@ The solver :ref:`settings API <ref_root>` is the main interface for controlling
     session_solver
     session
     system_coupling
-    warnings
+    warning
```

Comment:
```suggestion
    pyfluent_warnings
```

Reply:
Please replace it with `pyfluent_warnings`. 

`rename src/ansys/fluent/core/{warnings.py => pyfluent_warnings.py} (100%)`